featuretools/tests/primitive_tests/test_rolling_primitive.py

```diff
@@ -2,15 +2,17 @@
 
 import pandas as pd
 import pytest
+from numpy import isnan, nan
```

Comment:
I think if we are importing `pandas as pd` we should import `numpy as np` to be consistent.